src/screens/DetailScreen.js

```diff
@@ -159,7 +159,7 @@ DetailScreen.navigationOptions = ({ navigation, navigationOptions }) => {
     ),
     headerRight: (
       <WrapperRow>
-        {shareContent && (
+        {!!shareContent && (
```

Comment:
this was necessary because `shareContent` should be a text, that we do not want to be rendered here, but pass to the `openShare` method. it `shareContent` could happen that React Native wants to render it here because it is a string, which would result in an error on iOS as it is not wrapped in a Text component.

would be nice to have that change pointed out in the commit message.

Reply:
I will try to be more elaborate in future commits. :)